website/cue/reference/components/sinks/splunk_hec_logs.cue

```diff
@@ -141,6 +141,18 @@ components: sinks: splunk_hec_logs: {
 				examples: ["${SPLUNK_HEC_TOKEN}", "A94A8FE5CCB19BA61C4C08"]
 			}
 		}
+		timestamp_nanos_key: {
```

Comment:
I think `@timestamp.nanos` is something special, specific to Humio. I'd suggest we leave this configuration option off of the `splunk_hec_logs` sink.